leo/commands/setup.rs

```diff
@@ -14,82 +14,75 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{
-    cli::*,
-    cli_types::*,
-    commands::BuildCommand,
-    errors::{CLIError, RunError},
-};
+use crate::{commands::Command, context::Context};
+use anyhow::{anyhow, Result};
+
 use leo_compiler::{compiler::Compiler, group::targets::edwards_bls12::EdwardsGroupType};
 use leo_package::{
     outputs::{ProvingKeyFile, VerificationKeyFile},
-    root::Manifest,
     source::{MAIN_FILENAME, SOURCE_DIRECTORY_NAME},
 };
 
+use rand::thread_rng;
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates

Reply:
Here and below: added rustfmt rule for grouping the imports.